modules/machine-config-overview.adoc

```diff
@@ -50,7 +50,7 @@ The kinds of components that MCO can change include:
 
 [IMPORTANT]
 ====
-Changing SSH keys via machine configs is only supported for the `core` user.
+Changing SSH keys via machine configs is only supported for the `core` user. It is not supported to add any other user via machine configs.
```

Comment:
Maybe beyond the scope of this PR, since you're appending an admonition. So take this suggestion with a grain of salt. 

Is this note supposed to be part of the previously bullet point? ie: 

* users and groups: Change SSH keys in the passwd section post-installation.

(also, I know it's out of scope of this PR, but shouldn't that be password?) 

If so, and it is supposed to follow that bullet point, you might try adding a `+` above the [IMPORTANT] admonition so that it aligns with the previous bullet point. That said, it might not be necessary given that the following [IMPORTANT] admonition for FIPS does the same thing that this one currently does.

Additionally, you could swap "any other" for "another". They're fundamentally similar so take it how you will .

Otherwise, this LGTM.

Reply:
My understanding is: The bullet point explains that the only "users and gorups" allowed setting is to change the SSH keys (not password) and then it is the warning below which explains that only changing the `core` user is supported. As the note below is what clarifies what is **not** supported, I thought that would be the place to explain that it is also not supported to add users. Basically because the bullet point says what you can do, and the warning restricts what you cannot.

Does this make sense?